overlapping_panels/src/main/java/com/discord/panels/OverlappingPanelsLayout.kt

```diff
@@ -843,16 +843,16 @@ open class OverlappingPanelsLayout : FrameLayout {
   }
 
   private fun isTouchingCenterPanelWhileSidePanelOpen(event: MotionEvent): Boolean {
-    val rawX = event.rawX
+    val x = event.x
```

Comment:
The changes in this method fix the bug. The other changes in this PR are changes to the sample app to help reproduce the bug and verify that this fix works.